app/posts/apply-for-teacher-training/2023-03-17-telling-candidates-they-have-an-ske-condition.md

```diff
@@ -0,0 +1,123 @@
+
+From previous research, candidates often do not know what an SKE course is when they're asked to do one.
+
+Some of the main issues candidates have are not knowing:
+
+* why they have to do an SKE course
+* how to find an SKE course or who to do it with (SKE providers and ITT providers can be different)
+* SKE courses are free
+* they can get a bursary of £175 a week while they do their SKE course
+* how they can study their course (for example, online, in-person or both)
+
+## What we did
+
+After the training provider sends an offer to a candidate with an SKE condition, we show some guidance before a candidate accepts their offer. This helps explain what an SKE is, so the candidate can make an informed decision about accepting the offer or not.
+
+We give them information on:
+
+* the subject and length of their SKE course
+* why they have to do an SKE course
+* when they should complete their SKE course by
+* the SKE course being free and they'll get £175 a week while they do it
+* how they can study their SKE (online, in-person or both)
+
+![Screenshot showing the offer a candidate will recieve with a subject knowledge enhancement course attached. The course is for a 12 week biology course. The content also tells the user they should start their course by April 2023, there is a £175 bursary they can get while doing the course and there are options on how they can study which include full time, part time and online or in-person. The screen then asks the user to respond to the offer'.](ske-offer-page.png)
+
+### language SKE courses
+
+Sometimes training providers can ask candidates to do 2 language SKE courses if they’re applying to a modern language course. This is so they can have the ability to teach another language at their school if they need to.
+
+If there are 2 language SKE courses, we show slightly different content with grammatical changes so it makes sense to candidates.
+
+![Screenshot showing the offer a candidate will recieve with 2 language subject knowledge enhancement courses attached. The courses are for an 8 week French course and a 12 week Spanish course. The content also tells the user they should start their courses by April 2023, there is a £175 bursary they can get while doing the courses and there are options on how they can study which include full time, part time and online or in-person. The screen then asks the user to respond to the offer'](language-ske.png)
+
+Once they accept the offer, we show their SKE course with their other conditions, like their Disclosure and Barring Service (DBS) check.
+
+![Screenshot with the heading 'Study before you start training' followed by content saying the user has a 20 weeks maths course with the tag 'Not yet started'. This is followed by content explaining when the user should start their course, that there will be a £175 bursary for the course and to contact the University of Sussex if they have any issues.](offer-page.png)
```

Comment:
This bit doesn’t fit within the "Language SKE courses" section. Should it have its own subheading?